src/target/riscv/riscv.c

```diff
@@ -5353,6 +5373,16 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 		.usage = "[index ('on'|'off')]",
 		.help = "Controls which RISC-V triggers shall not be touched by OpenOCD.",
 	},
+	{
+		.name = "mmu_translation_driver",
+		.handler = handle_riscv_mmu_translation_driver,
+		.mode = COMMAND_ANY,
+		.usage = "'sw'|'hw'|'off'",
+		.help = "Configure address translation way: "
```

Comment:
```suggestion
		.help = "Configure the virtual address translation mode: "
```

or

```suggestion
		.help = "Configure the virtual address translation mechanism: "
```

or similar

Reply:
Addressed